src/Response/Exception/InvalidZipResponseException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+declare(strict_types=1);
+
+namespace MarcusJaschen\Collmex\Response\Exception;
+
+/**
+ * This exception is thrown if an invalid Zip response is
+ * detected, i. e. when a Zip Response doesn't contain
```

Comment:
There needs to be a comma after the i.e., but not space after the i.

and: doesn't -> does not

Reply:
Update incoming…